### DISCUSSION TITLE
Welcome to amazon-eks-one-click-cluster-upgrade Discussions!

### DISCUSSION THREAD

**Discussion by bryantbiggs** [aws-samples/eks-cluster-upgrade#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mubarak-j** [comment]
Thank you for reviving this project 🚀
